### PR TITLE
Release notes for 3.4.34

### PR DESCRIPTION
**Pull request by timtheisen**

### CODE REVIEW

File: docs/release/3.4/release-3-4-34.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+
+Summary of changes
+------------------
+
+This release contains:
+
+-   XCache 1.1.1: ATLAS and CMS XCache; Stash Origin HTTP/S Support
+-   osg-configure 2.4: improvements for Slurm, PBS Pro, BOSCO, Frontier Squid
+-   osg-xrootd 3.4-4: New meta package for OSG XRootD configuration
+-   xrootd-hdfs 2.1.6: now includes default XRootD configuration in /etc
```
Comment (brianhlin) [inline docs/release/3.4/release-3-4-34.md]:
```suggestion
-   XRootD HDFS 2.1.6: includes default configuration in `/etc/xrootd/40-xrootd-hdfs.cfg`.
```

File: docs/release/3.4/release-3-4-34.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+-   xrootd-lcmaps 1.7.4: now includes default XRootD configuration in /etc
```
Comment (brianhlin) [inline docs/release/3.4/release-3-4-34.md]:
```suggestion
-   XRootD LCMAPS 1.7.4: includes default authorization configuration in `/etc/xrootd/config.d/40-xrootd-lcmaps.cfg`.
    To use the default configuration, uncomment the `# set EnableLcmaps = 1` line in `/etc/xrootd/config.d/10-xrootd-lcmaps.cfg`.
```

File: docs/release/3.4/release-3-4-34.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+**Release Date**: 2019-08-29
+
+Summary of changes
+------------------
+
+This release contains:
+
+-   XCache 1.1.1: ATLAS and CMS XCache; Stash Origin HTTP/S Support
+-   osg-configure 2.4: improvements for Slurm, PBS Pro, BOSCO, Frontier Squid
+-   osg-xrootd 3.4-4: New meta package for OSG XRootD configuration
```
Comment (brianhlin) [inline docs/release/3.4/release-3-4-34.md]:
```suggestion
-   OSG XRootD 3.5: A meta-package including common configuration across [standalone](/data/xrootd/install-standalone),
    [storage element](/data/xrootd/install-storage-element), and [caching](/data/stashcache/overview) installations of
    XRootD.
```

File: docs/release/3.4/release-3-4-34.md [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+===========================
+
+**Release Date**: 2019-08-29
+
+Summary of changes
+------------------
+
+This release contains:
+
+-   XCache 1.1.1: ATLAS and CMS XCache; Stash Origin HTTP/S Support
```
Comment (brianhlin) [inline docs/release/3.4/release-3-4-34.md]:
```suggestion
-   XCache 1.1.1: This release includes packages for ATLAS and CMS XCaches as well as Stash Origin HTTP/S support.
```

File: docs/release/3.4/release-3-4-34.md [hunk 5]
```diff
@@ -0,0 +1,215 @@
+OSG Software Release 3.4.34
+===========================
+
+**Release Date**: 2019-08-29
```
Comment (brianhlin) [inline docs/release/3.4/release-3-4-34.md]:
```suggestion
**Release Date**: 2019-08-29    
**Supported OS Versions:** EL7, EL6
```

File: docs/release/3.4/release-3-4-34.md [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+
+Known Issues
+------------
+
+None.
+
+Updating to the new release
+---------------------------
+
+To update to the OSG 3.4 series, please consult the page on [updating between release series](/release/release_series#updating-from-osg-31-32-33-to-33-or-34).
```
Comment (brianhlin) [inline docs/release/3.4/release-3-4-34.md]:
```suggestion
```

File: docs/release/3.4/release-3-4-34.md [hunk 7]
```diff
@@ -0,0 +1,218 @@
...
+**Release Date**: 2019-08-29    
+**Supported OS Versions:** EL7, EL6
+
+Summary of changes
+------------------
+
+This release contains:
+
+-   XCache 1.1.1: This release includes packages for ATLAS and CMS XCaches as well as Stash Origin HTTP/S support.
+-   osg-configure 2.4: improvements for Slurm, PBS Pro, BOSCO, Frontier Squid
```
Comment (brianhlin) [inline docs/release/3.4/release-3-4-34.md]:
```suggestion
-   OSG Configure 2.4: improvements for Slurm, PBS Pro, BOSCO, Frontier Squid
```

File: docs/release/3.4/release-3-4-34.md [hunk 8]
```diff
@@ -0,0 +1,218 @@
...
+-   OSG XRootD 3.5: A meta-package including common configuration across [standalone](/data/xrootd/install-standalone),
+    [storage element](/data/xrootd/install-storage-element), and [caching](/data/stashcache/overview) installations of
+    XRootD.
+-   XRootD HDFS 2.1.6: includes default configuration in `/etc/xrootd/40-xrootd-hdfs.cfg`.
+-   XRootD LCMAPS 1.7.4: includes default authorization configuration in `/etc/xrootd/config.d/40-xrootd-lcmaps.cfg`.
+    To use the default configuration, uncomment the `# set EnableLcmaps = 1` line in `/etc/xrootd/config.d/10-xrootd-lcmaps.cfg`.
+-   MyProxy 6.2.4: Remove usage statistics collection support
+-   [CCTools 7.0.14](http://ccl.cse.nd.edu/software/): Bug fix release
+-   osg-system-profiler 1.4.3: Remove collection of obsolete information
```
Comment (brianhlin) [inline docs/release/3.4/release-3-4-34.md]:
```suggestion
-   OSG System Profiler 1.4.3: Remove collection of obsolete information
```

File: docs/release/3.4/release-3-4-34.md (hunk 7 above)
Comment (timtheisen) [reply docs/release/3.4/release-3-4-34.md]: Done offline

File: docs/release/3.4/release-3-4-34.md [hunk 9]
```diff
@@ -0,0 +1,218 @@
...
+**Release Date**: 2019-08-29    
+**Supported OS Versions:** EL7, EL6
+
+Summary of changes
+------------------
+
+This release contains:
+
+-   XCache 1.1.1: This release includes packages for ATLAS and CMS XCaches as well as Stash Origin HTTP/S support.
+-   osg-configure 2.4.1: improvements for Slurm, PBS Pro, BOSCO, Frontier Squid
```
Comment (brianhlin) [inline docs/release/3.4/release-3-4-34.md]: I'd still like to see the formatting change of `OSG Configure` to match everything else

```suggestion
-   OSG Configure 2.4.1: improvements for Slurm, PBS Pro, BOSCO, Frontier Squid
```

Comment (timtheisen) [reply docs/release/3.4/release-3-4-34.md]: Resolve offline

**Pull request merged**